stm32mp1.mk

```diff
@@ -94,6 +100,18 @@ include toolchain.mk
 ################################################################################
 # OP-TEE OS
 ################################################################################
+ifeq ($(WITH_RPMB_TEST),y)
+CFG_RPMB_FS_DEV_ID ?= 1
+OPTEE_OS_COMMON_FLAGS += \
+		CFG_RPMB_FS_DEV_ID=$(CFG_RPMB_FS_DEV_ID) \
+		CFG_RPMB_FS=y \
+		CFG_RPMB_TESTKEY=y \
+		CFG_REE_FS_ALLOW_RESET=y
+$(info -------------------------------------------------------------------------------------)
+$(info "WARNING: building with RPMB support for test purpose")
```

Comment:
Why warn? Because of `CFG_REE_FS_ALLOW_RESET=y`? I suggest being more explicit.

Reply:
Because of `CFG_RPMB_TESTKEY=y`